### ISSUE TITLE
Problem with KHR_lights_punctual value, when KTX2 compressing using the gltf-transform cli tool

### ISSUE THREAD

**Issue by gtolarc** [donmccurdy/glTF-Transform#483]
When compressing KTX2 using the gltf-transform cli tool, the innerConeAngle/outerConeAngle values of KHR_lights_punctual are initialized to default values. I think this is a bug, check it please.

```bash
gltf-transform etc1s sample.gltf sample_out.gltf
```
FROM
<img width="369" alt="Screen Shot 2022-02-07 at 3 04 13 AM" src="https://user-images.githubusercontent.com/8623798/152694750-cb58a1d3-a2a7-49ba-8824-d5c012c637cb.png">

TO
<img width="336" alt="Screen Shot 2022-02-07 at 3 04 32 AM" src="https://user-images.githubusercontent.com/8623798/152694759-819eb75d-985b-4237-8587-ce5d1b1ada26.png">

**Comment by gtolarc** [comment]
Perhaps it was related to this.
https://github.com/donmccurdy/glTF-Transform/blob/main/packages/extensions/src/khr-lights-punctual/light.ts#L47

**Comment by donmccurdy** [comment]
Hi! Could you check if the same thing happens when running:

```

gltf-transform copy sample.gltf sample_out.gltf

```

**Comment by donmccurdy** [comment]
Found the cause – will be fixed in #484. Thank you!

**Comment by gtolarc** [comment]
@donmccurdy When can it be distributed? It would be nice if I could try it out sooner.

**Comment by donmccurdy** [comment]
Published as v2.0.1 👍
